libraries/botbuilder-dialogs-adaptive/src/adaptiveDialogState.ts

```diff
@@ -0,0 +1,5 @@
+import { ActionState } from "./sequenceContext";
+
+export class AdaptiveDialogState {
```

Comment:
>AdaptiveDialogState [](start = 13, length = 19)

again... this should just be an interface like I had it.

Reply:
Got it, rolled back to interface.